build.rs

```diff
@@ -97,8 +97,10 @@ fn main() {
         println!("cargo:rustc-cfg=libc_thread_local");
     }
 
-    // Rust >= 1.62.0 allows to use `const_extern_fn` for "Rust" and "C".
-    if rustc_minor_ver >= 62 {
+    // Rust > 1.62.0 allows to use `const_extern_fn` for "Rust" and "C".
+    // Nightlies before 2022-04-18 (partly through 1.62 nightlies) don't
+    // have the feature, so we require 1.63.
+    if (!is_nightly && rustc_minor_ver >= 62) || (is_nightly && rustc_minor_ver >= 63) {
```

Comment:
This will break the 1.62 nightly vertsions after that feature was stabilized instead, right? Then I think it's better to handle with the nightly date.

Reply:
Here's the related code:
https://github.com/rust-lang/libc/blob/ad3ce341e3bad465a6672517fa952910f0a5a64f/build.rs#L115-L152

If it's nightly, the output would be something like `rustc 1.68.0-nightly (0442fbabe 2023-01-10)` and we could get the date on parsing.

Reply:
I don't want to get into date parsing. This won't "break" versions after that nightly, those just won't use the `const extern` feature.

Reply:
Why? And it disables the cfg for _newer_ 1.62 nightly users.